docs/extending.md

```diff
@@ -0,0 +1,19 @@
+# Extending the adapter/serializer
```

Comment:
The extending section from the README needs to be remove because it's no longer accurate.